decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -74,18 +74,29 @@ def external_icon(path, options = {})
       classes = _icon_classes(options) + ["external-icon"]
 
       if path.split(".").last == "svg"
+        icon_path = application_path(path)
+        return unless icon_path
+
         attributes = { class: classes.join(" ") }.merge(options)
-        asset = File.read(application_path(path))
+        asset = File.read(icon_path)
         asset.gsub("<svg ", "<svg#{tag_builder.tag_options(attributes)} ").html_safe
```

Comment:
CodeQL has detected a possible vulnerability here: 

![image](https://github.com/decidim/decidim/assets/717367/f11f6cd8-3703-447a-895b-e4c59935cc12)


Can you check it please? Thanks

Reply:
To be honest, this is a false positive from  my point of view.  In that particular place, we are reading a svg that we provide from a file, then we alter its attributes to set classes and other attributes we need. 

I would suggest to ignore this, and mark it accordingly.